src/Symfony/Component/DependencyInjection/DefinitionDecorator.php

```diff
@@ -99,7 +99,7 @@ public function setFactoryMethod($method)
     /**
      * {@inheritdoc}
      */
-    public function setFactoryService($service)
+    public function setFactoryService($service, $triggerDeprecationError = true)
     {
         $this->changes['factory_service'] = true;
 
```

Comment:
Oh good catch, we never call this internally so this went unnoticed. Would you like to send a fix?